Distinct/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Distinct;
+
+public static class DistinctClass
+{
+    public static SolutionDistinct SolutionInstance = new SolutionDistinct();
```

Comment:
space between proprietes and you can just use initialize it with new
```suggestion
    public static SolutionDistinct SolutionInstance = new();
```